ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/Types.hs

```diff
@@ -129,8 +129,9 @@ sameUnexpectedError e1 e2 = case (e1, e2) of
 newtype BlockSize = BlockSize {unBlockSize :: Word64}
     deriving (Show, Generic, NoUnexpectedThunks)
 
--- | The offset at which a block is stored in a file.
-type BlockOffset = Word64
+-- | In case of a parser error, this is the offset at which the file should be
+-- truncated.
+type TruncateOffset = Word64
```

Comment:
CBOR uses `Word64` for this offset and it seemed weird to go and wrap them with `AbsOffset`. I also think using a different name for this is a bit more expressive.